From a7f31c843ec2eed4c5a63679775e2f20ff552244 Mon Sep 17 00:00:00 2001
From: Jacob Martin <jjm_223@hotmail.com>
Date: Tue, 27 Dec 2016 19:00:26 -0600
Subject: [PATCH 08/12] Don't include banner base color when comparing
 metadata.

1.11 introduced base colors being stored by item damage value.
---
 src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaBanner.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaBanner.java b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaBanner.java
index aa1a210..f5ad80a 100644
--- a/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaBanner.java
+++ b/src/main/java/org/bukkit/craftbukkit/inventory/CraftMetaBanner.java
@@ -181,19 +181,19 @@ public class CraftMetaBanner extends CraftMetaItem implements BannerMeta {
         if (meta instanceof CraftMetaBanner) {
             CraftMetaBanner that = (CraftMetaBanner) meta;
 
-            return base == that.base && patterns.equals(that.patterns);
+            return patterns.equals(that.patterns);
         }
         return true;
     }
 
     @Override
     boolean notUncommon(CraftMetaItem meta) {
-        return super.notUncommon(meta) && (meta instanceof CraftMetaBanner || (patterns.isEmpty() && base == null));
+        return super.notUncommon(meta) && (meta instanceof CraftMetaBanner || patterns.isEmpty());
     }
 
     @Override
     boolean isEmpty() {
-        return super.isEmpty() && patterns.isEmpty() && base == null;
+        return super.isEmpty() && patterns.isEmpty();
     }
 
     @Override
-- 
1.9.1

